### PR TITLE
Feat: Home Page

### PR DESCRIPTION
**Pull request by Shivanshsharma01**
Added home page and did some path change

https://user-images.githubusercontent.com/86787002/154920016-f6670964-0246-4486-aec9-8cc520039df9.mp4

### CODE REVIEW

File: Page/wishlist/wishlist.html [hunk 1]
```diff
@@ -24,110 +24,109 @@
...
-                            </span>
-                            <span class="discount">
-                                (10% OFF)
-                            </span>
+    </header>
+    <main>
+        <div class="wishlist-container">
+            <div class="wishlist-heading">My Wishlist</div>
+            <div class="wishlist-card">
+                <div class="card .price-card">
```
Comment (nareshpoonia) [inline Page/wishlist/wishlist.html]: Shouldn't the class name be price-card, instead of .price-card

File: Page/wishlist/wishlist.html [hunk 2]
```diff
@@ -24,110 +24,109 @@
...
+                    <div class="product-image">
+                        <img src="https://images.unsplash.com/photo-1612724189298-89d36b10b26d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1025&q=80"
+                            alt="Image" />
```
Comment (nareshpoonia) [inline Page/wishlist/wishlist.html]: You might consider using Alt Text as Sneakers instead of Image

**Review by nareshpoonia — APPROVED** [review]
Just a few comments, otherwise code looks good to merge

File: Page/wishlist/wishlist.html (hunk 1 above)
Comment (Shivanshsharma01) [reply Page/wishlist/wishlist.html]: yess

File: Page/wishlist/wishlist.html (hunk 2 above)
Comment (Shivanshsharma01) [reply Page/wishlist/wishlist.html]: sure

**Review by GuptaEesh — APPROVED** [review]
All the necessary changes have already been listed.
The code looks good to merge.

**Pull request merged**